package.json

```diff
@@ -152,6 +152,14 @@
     "rimraf": "^3.0.2",
     "semistandard": "^16.0.1"
   },
+  "peerDependencies": { 
+    "node-gyp": "*"
+  },
+  "peerDependenciesMeta": {
+    "node-gyp": {
+      "optional": true
```

Comment:
I recall some versions of npm auto-installing these erroneously - are you aware of any remaining problems this might cause?

Reply:
Worst case, you'll get a message that the peer dependency is missing?